crates/bevy_render/src/settings.rs

```diff
@@ -47,6 +47,8 @@ impl Default for WgpuSettings {
     fn default() -> Self {
         let default_backends = if cfg!(feature = "webgl") {
             Backends::GL
+        } else if cfg!(windows) {
```

Comment:
Can we leave a TODO comment linking the wgpu issue to ensure this gets changed back once it's been addressed by them?